fe/fe-core/src/main/java/com/starrocks/analysis/CreateUserStmt.java

```diff
@@ -53,6 +41,7 @@
  * 3. create user user@xx [identified by 'password' | identified with auth_plugin [AS | BY auth_string]] role role_name
  *      not only create the specified user, but also grant all privs of the specified role to the user.
  */
+// TODO: CreateUserStmt and AlterUserStmt should share the same parameter and check logic with BaseCreateAlterUserStmt
```

Comment:
How about completing this TODO in `PrivilegeStmtAnalyzer` right now? Since you have already made impressive progress in refactoring some logic about user & password analysis. :-)

Reply:
done